src/parsing/syntax.ml

```diff
@@ -212,6 +213,18 @@ type p_tactic_aux =
 
 type p_tactic = p_tactic_aux loc
 
+type p_tactic_tree = Tactic of p_tactic option * p_tactic_tree list;;
+
+(**tree_to_list tactic_tree returns the list of tactics
+  described by the tree*)
```

Comment:
Thanks to try to follow the style of comments used in the other files. You should instead write something like: `(** [tac_list_of_tact_tree t] converts the tactic tree [t] into a list of tactics by doing a depth-first search. *)`.